backend/attendance/README.md

```
@@ -16,14 +13,26 @@ is the function of the muster roll service.
+## Key configuration (Payments V2)
+
+- `attendance.register.kafka.muster.status.update.topic`: Topic consumed to sync period statuses from muster-roll.
+- `egov.expense.calculator.host` + `egov.expense.calculator.billing.period.search.endpoint`: Billing period metadata lookup.
+- `egov.muster.roll.host` + `egov.muster.roll.search.endpoint`: Fallback muster-roll search when a period status is absent.
 
-### API Specs
+## API Specs
 
-https://raw.githubusercontent.com/egovernments/DIGIT-Specs/master/Domain%20Services/Works/Attendance-Service-v1.0.0.yaml
+- Payments V2 contract: `attendance/Attendance-Service-2.0.0.yaml`
+- Legacy reference: https://raw.githubusercontent.com/egovernments/DIGIT-Specs/master/Domain%20Services/Works/Attendance-Service-v1.0.0.yaml
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)